src/main/java/rx/internal/operators/BackpressureUtils.java

```diff
@@ -45,7 +45,9 @@ private BackpressureUtils() {
      * @param n
      *            the number of requests to add to the requested count
      * @return requested value just prior to successful addition
+     * @deprecated Android has issues with reflection-based atomics
```

Comment:
Yeah I'm not sure. I would imagine the new runtime on Android 5.0+ could do something about it. I'll have to investigate it more.